### ISSUE TITLE
Named DataFrame indices shouldn't introduce an empty row between header and data

### ISSUE THREAD

**Issue by fzumstein** [microsoft/python-in-excel#43]
When showing Values of a DataFrame with a named index, it introduces an empty row between header and data. There's no reason for it though, and `pd.to_excel()` doesn't do it either (even if there's additionally a `df.columns.name`!). So currently, you'd have to do `df.reset_index()` to format it correctly.

**Comment by KimJun9011** [comment]
![df - with extra space](https://github.com/microsoft/python-in-excel/assets/148293307/f08cb754-e9a0-4cf5-b11b-7649bd3e2585)
If you try the same example on standard jupyter environment, you will see the exact same space allocated. I do not see this as an issue.

**Comment by fzumstein** [comment]
That's right, because Jupyter notebooks leave space for a potential columns name, but that concept doesn't really translate to Excel (see e.g., Excel tables).
